synnax/pkg/service/framer/service.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+package framer
+
+import (
+	"context"
+
+	"github.com/synnaxlabs/synnax/pkg/distribution/framer"
+	"github.com/synnaxlabs/synnax/pkg/service/framer/downsampler"
+)
+
+type Service struct {
+	Internal *framer.Service // distribution layer frame service
+}
+
+// got these from api/framer.go calls to Internal
```

Comment:
Is this comment necessary? If so, make sure that they adhere to the go doc standard of putting the method name as the first word in the comment.